docs/tutorials/PickupGuide.md

```diff
@@ -0,0 +1,77 @@
+PickupGuide
+===========
+
+### From open.mp Wiki
+
+  
+
+
+### Define the pickupid
+
+The first thing to be done when creating pickups is creating a place to store their ID. This will be done in a global variable so it can be set when you create the pickup and read when you pick up a pickup, calling a callback with the ID of the pickup you picked up. For this example we will use the name "mypickup".
+
+new mypickup;
```

Comment:
`new MyPickup;`

Reply:
bro we have Functions named in PascalCase , so we should not make our variable PascalCase . 
also in pawno its up to you

Reply:
Your PR probably won't be accepted if it doesn't abide by the Pawn Curriculum. Also it's Pawn.

Reply:
Globals use `g` as a prefix according to the style guide.

Reply:
Also this should be in a code block:

```suggestion
\`\`\`
new mypickup;
\`\`\`
```